### PR TITLE
refactor(routesCdnPath): FRMW-380 convert to quark

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-380

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `configs.routesCdnPath` to Quark Service.

### CODE REVIEW

**Comment by 100stacks** [comment]
This PR requires https://github.com/rackerlabs/encore-ui/pull/1496 be merged first to complete `routesCdnPath` refactoring.  This PR removes the `encore.ui.configs` component.

**Comment by 100stacks** [comment]
## `routesCdnPath` Quark Service

### Demo Page
screen shot 2016-01-19 at 9 15 55 am

### API Docs
screen shot 2016-01-19 at 9 16 13 am

File: src/quarks/routesCdnPath/routesCdnPath.meta.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "routesCdnPath",
+    "stability": "stable",
+    "description": "Allows users to override the URL used when in a local/staging environment.",
```
Comment (parlarjb) [inline src/quarks/routesCdnPath/routesCdnPath.meta.json]: This description is not correct. The main point of `routesCdnPath` is to provide the CDN locations from which to load the nav JSON. The "override" bit has to do with this being a `$provider`, meaning we let people override when doing local dev.

Comment (CITguy) [inline src/quarks/routesCdnPath/routesCdnPath.meta.json]: @100stacks, please leave description blank for now.

**Comment by 100stacks** [comment]
@parlarjb @CITguy, updated description. :+1:

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**